docs/getting-started/install-resoto/kubernetes.md

```diff
@@ -51,19 +51,19 @@ Wait until the ArangoDB deployment is ready. You can check the conditions in the
 $ kubectl wait --for=condition=ready arangodeployment/single-server
 ```
 
-### Create Helm Values File
+### Configure Helm Values File
```

Comment:
We already have a default helm values file created which in turn is being modified by the user not being created from scratch.

Reply:
This guide is suggesting that users create a "barebones" `resoto-values.yaml` file, instead of modifying the `values.yaml`.

Reply:
But is there a need for user to create the file, given the changes which we are telling the user to make in that are already present as defaults?

Reply:
They would need to fetch the file, in that case. Either way, it will be instructions to create the file, whether it be from the repository or from scratch. (We should add a `curl` command below if we want users to fetch a copy from the repo.)

Reply:
They wouldn't need to fetch it as the defaults would be fetched via the `helm install` itself. If I install arango & just do `helm install resoto resoto/resoto --set image.tag=latest`, my resoto will point to default arango installation settings. Or am I missing some usecase over here 😅

Reply:
`helm install` is not executed until later. I believe the intention here was to configure the values file first and supply it on install.

Reply:
Why I even need to fetch the values file when I can list all the helm defaults as well override everything via `--set` or providing a `<<EOF` during my installation. Ideally `Getting Started` for any project means to get the project up & running with the default and minimal configuration, and if we expect the user to create the values.yaml then we should have a detailed section about `How to configure Resoto with Kubernetes`

Reply:
I'm not disagreeing with you here, but your changes do not make sense unless we change the ordering/structure of this guide -- you ask users to edit a file that does not yet exist. 😉 Let's make the small changes to point to the new repo first, then we can discuss the best set of ordered instructions to get users up and running. I do think that it is much more user-friendly for users to edit an existing file than to create a new one.

Reply:
Agreed, any amount of documentation is not more than enough for each and every user 😅